src/models/User.ts

```diff
@@ -1,5 +1,25 @@
 export interface User {
-    name: string,
-    email: string,
+    name: string
+    email: string
+    admin: boolean
     color?: string
-}
\ No newline at end of file
+}
+
```

Comment:
Our project seems to have a pattern for keeping these map functions close to the model interface. It is done on both bookings and checkins models. The `parseUser` function was renamed into this `mapAuthResponseToUser` and another `mapResponseToUser` was created as needed